editor/code/package.json

```diff
@@ -75,33 +75,20 @@
     ],
     "configuration": [
       {
+        "title": "General",
```

Comment:
Maybe "Proof Checking" or "Error Recovery" or maybe "Proof Checking and Error Recovery"?